torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -864,7 +864,8 @@ struct PythonPrintImpl {
 
   void printConstant(TaggedStringStream& stmt, const IValue& v) {
     const auto customFormatter = [&](std::ostream& ss, const IValue& v) {
-      if (v.isTensor() || containsNonASCIIString(v)) {
+      if (v.isTensor() || containsNonASCIIString(v) ||
+          (v.isObject() && !v.type()->is_module())) {
```

Comment:
Should we also handle Function objects here?

Reply:
not sure, why would we?

Reply:
Yea, don't think that's needed. Could consider using https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/ivalue.cpp#L125 to check for contained object constants, although i suspect that's somewhat unlikely.

Reply:
By handling I actually meant not handling it 😀, but bailing out if we see them (similarly to how bail out for modules here).